### DISCUSSION TITLE
Customise path to .so file in macOS App bundle.

### DISCUSSION THREAD

**Discussion by TilmanK** [pyinstaller/pyinstaller#8123] [Help]
I'm running into a problem that occurs since PyInstaller Version 6. I'm using PyInstaller to create an app bundle for macOS. That app bundle is created as expected and works fine. Nevertheless, I need to pass it to an external process for signing that fails, because it expects one of my .so files in a different folder. The file is currently located in `Frameworks` folder of the bundle but should be in a subfolder (e.g., `Frameworks/sub`). Is there any way to tell PyInstaller to do this?

Neither the documentation nor any Googling has given me a hint. Any help is appreciated. Thanks! :)

**Comment by rokm** [comment]
What .so file is that, where does it come from, and why does it need to be in a sub-directory? If this is discovered via binary dependency analysis, then under PyInstaller 5.x and earlier, that would be collected into `MacOS` directory without sub-directory as well (i.e, not `MacOS/sub`).
